std_msgs/Header header
std_msgs/Int8 mode
std_msgs/String gear
std_msgs/Int8 signal
std_msgs/Float32 velocity
std_msgs/Float32 heading
geometry_msgs/Point position
